app/controllers/venues_controller.rb

```diff
@@ -25,6 +27,29 @@ def create
     end
   end
 
+  def edit
+    @venue = Venue.find(params[:id])
+  end
+
+  def update
+    updated_params = venue_params
+    updated_params[:user_id] = current_user.id
+    @venue = Venue.find(params[:id])
+    @venue.update(updated_params)
+    if @venue.save
```

Comment:
Delete lines 35-39, write

```ruby
@venue  = Venue.find(params[:id])
if @venue.update(venue_params)
```

Again, I wouldn't re-update the current_user, that could lead to dangerous behavior